core/src/test/java/com/linecorp/armeria/common/RequestContextTest.java

```diff
@@ -307,6 +308,83 @@ public void makeContextAwareCompletableFutureWithAsyncChaining() throws Exceptio
         }
     }
 
+    @Test
+    public void minimalCompletionStageUsingCompleteAsync() throws Exception {
+        final RequestContext context = createContext(false);
+        final CompletableFuture<Integer> originalFuture = new CompletableFuture<>();
+        @SuppressWarnings("unchecked")
+        final RequestContextAwareCompletableFuture<Integer> contextAwareFuture =
+                (RequestContextAwareCompletableFuture) context.makeContextAware(originalFuture);
+        final CompletionStage<Integer> completionStage = contextAwareFuture.minimalCompletionStage();
+
+        contextAwareFuture.completeAsync(() -> {
+            throw new RuntimeException();
+        });
+        contextAwareFuture.complete(1);
```

Comment:
What is this test for?
If we add `Thread.sleep(1000)` before this line the test result is totally different.
If we want to test `completeAsync` we need to remove this `contextAwareFuture.complete(1);`.
And we need to change all the `assertThat(...)` below.

Reply:
Thank you for the feedback.
It was a test on the `toCompletableFuture()` method call when converting to minimalStage. I deleted the unnecessary code and changed it to make sense.